test/e2e/aro.go

```diff
@@ -0,0 +1,79 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/util/azureclient"
+)
+
+const (
+	wSsh = "99-worker-ssh"
+	mSsh = "99-master-ssh"
+)
+
+// The high level idea of these tests is to definitevily say that a cluster is an ARO cluster.
+var _ = Describe("Verify Attributes of ARO Cluster", func() {
+	ctx := context.Background()
+	// acrDomainList should contain acrDomain verifier
+	acrDomainList := []string{"arointsvc.azurecr.io", "arointsvc.azurecr.us", "arosvc.azurecr.io", "arosvc.azurecr.us"}
+	azEnvironmentList := []string{azureclient.PublicCloud.Environment.Name, azureclient.USGovernmentCloud.Environment.Name}
+	It("should be possible to definitely confirm cluster is ARO", func() {
```

Comment:
nit: not sure if you meant definitively here
```suggestion
	It("should be possible to definitively confirm cluster is ARO", func() {
```